lib/adiwg/mdtranslator/readers/iso19115_2/modules/module_citation.rb

```diff
@@ -69,6 +72,11 @@ def self.unpack(xCitationParent, hResponseObj)
                   hCitation[:responsibleParties] =
                      AdiwgUtils.consolidate_iso191152_rparties(hCitation[:responsibleParties]).compact
 
+                  # onlineResource (optional)
+                  # <xs:element name="onlineResource" type="gmd:CI_OnlineResource_PropertyType" minOccurs="0"/>
+                  xOnlineResource = xCitation.xpath(@@onlineResourceXPath)[0]
```

Comment:
since we can have >1 responsible parties associated with a citation let's get every onlineResource instead of just the first one.

Reply:
this is what i'm seeing for `CI_Citation_Type`
```xml 
<xs:element name="citedResponsibleParty" type="gmd:CI_ResponsibleParty_PropertyType" minOccurs="0" maxOccurs="unbounded"/>
```

Reply:
aha, i see the confusion, the -2 spec doesn't align with how dcat writer expects things to be set up, but the -3 does. 

this is a great question for jonathan as the -2 writer expects things configured based on the spec.